.eslintrc.js

```diff
@@ -299,12 +299,14 @@ module.exports = {
     BigUint64Array: 'readable',
     Event: 'readable',
     EventTarget: 'readable',
+    FinalizationRegistry: 'readable',
     MessageChannel: 'readable',
     MessageEvent: 'readable',
     MessagePort: 'readable',
     TextEncoder: 'readable',
     TextDecoder: 'readable',
     queueMicrotask: 'readable',
+    WeakRef: 'readable',
```

Comment:
This are already used in the code, but never as globals

https://github.com/nodejs/node/blob/9df3b762d770789a6260e4dfbc11451ef62a9cec/lib/internal/util/iterable_weak_map.js#L11-L21

Maybe this PR should used them in a similar fashion?

Reply:
I've changed the code to use them as globals - when they move to primordials I'll grab them from there.